python-pytest.el

```diff
@@ -127,6 +127,16 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
            (set-default symbol value)
            value))))
 
+(defcustom python-pytest-use-treesit nil
```

Comment:
How about 
```suggestion
(defcustom python-pytest-use-treesit (featurep 'treesit)
```

Reply:
is there any reason to NOT use treesitter even though it's available? i'd rather have less configuration and more ‘dwim’ behaviour, and not having this customizable at all would have my preference.